providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/grpc.py

```diff
@@ -37,30 +37,32 @@
 
 
 class GrpcResolver:
-    MAX_BACK_OFF = 120
-
     def __init__(
         self,
         config: Config,
         emit_provider_ready: typing.Callable[[ProviderEventDetails], None],
         emit_provider_error: typing.Callable[[ProviderEventDetails], None],
+        emit_provider_stale: typing.Callable[[ProviderEventDetails], None],
         emit_provider_configuration_changed: typing.Callable[
             [ProviderEventDetails], None
         ],
     ):
         self.config = config
         self.emit_provider_ready = emit_provider_ready
         self.emit_provider_error = emit_provider_error
+        self.emit_provider_stale = emit_provider_stale
         self.emit_provider_configuration_changed = emit_provider_configuration_changed
         self.cache: typing.Optional[BaseCacheImpl] = (
             LRUCache(maxsize=self.config.max_cache_size)
-            if self.config.cache_type == CacheType.LRU
+            if self.config.cache == CacheType.LRU
             else None
         )
         self.stub, self.channel = self._create_stub()
         self.retry_backoff_seconds = config.retry_backoff_ms * 0.001
+        self.retry_backoff_max_seconds = config.retry_backoff_ms * 0.001
```

Comment:
why are there now 2 attributes with the same value? did you mean to use `config.retry_backoff_max_ms` instead?

Reply:
Nice catch thank you

Reply:
fixed config property usage in https://github.com/open-feature/python-sdk-contrib/pull/117/commits/38cf9b99cf04e1b9194f1738f7272fa1710db9b1